charts/plex-media-server/values.yaml

```
@@ -36,6 +36,14 @@ pms:
   # NOTE: When set, 'configStorage' and 'storageClassName' are ignored.
   configExistingClaim: ""
 
+  # Enabling this will add nvidia.com/gpu: 1 to limits, will set
+  # environment for the nvidia operator, and will mount {device} into
+  # the container (default /dev/nvidia0)
+  gpu:
+    nvidia:
+      enabled: false
+      # device: /dev/nvidia0
```

Would you mind leaving this uncommented as the default value to stay consistent with the rest of the configurations in the file. 


```suggestion
      device: /dev/nvidia0
```

---

Hi @MarshallAsch in the latest version I removed it as it's unnecessary with the nvidia operator. Old habits you could say. I will re-generate the documentation, thank you for the review!